tests/src/test/scala/cats/tests/CokleisliTests.scala

```diff
@@ -11,6 +11,9 @@ import org.scalacheck.Arbitrary
 
 class CokleisliTests extends SlowCatsSuite {
 
+  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
+    slowCheckConfiguration.copy(sizeRange = slowCheckConfiguration.sizeRange.min(5))
```

Comment:
Reducing `minSuccessful` would reduce the time for `CokleisliTests`:

- 50, around 1m20s (now, default value `CatsSuite` for JVM)
- 25, around 50s
- 20, around 20s

Opinions?